app/channels/activity_channel.rb

```diff
@@ -19,34 +18,43 @@ def redis
   def self.subscribed_ids
     value = connection_for_redis.get('SUBSCRIBED_IDS')
     return [] unless value
+
     JSON.parse(value)
   end
 
   def receive(data)
-    process_asset_group(strong_params_for_asset_group(data)) if (data["asset_group"])
-    process_activity(strong_params_for_activity(data)) if (data["activity"])
```

Comment:
Lots of this, not sure if this was Rubocop removing the unnecessary brackets or me. There are links to commit ranges that exclude the basic Rubocop formatting stuff to make this a bit easier to follow.